### ISSUE TITLE
[Tooltip]: make tooltip accessible from keyboard

### ISSUE THREAD

**Issue by AlekseyManetov** [epam/UUI#2322]
## To Do
According the [W3C guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/) tooltip content should be displayed when tooltip target receives keyboard focus and should close by focus lose. 
See the example [here](https://zoebijl.github.io/apg-tooltip/)

## Describe the solution you'd like (optional)

Let's investigate how we can apply this logic for our Tooltip and components which can be used with it. We should decide how to make tooltip target component focusable, probably we should provide empty `onClick` handler in `renderTarget` params, but not all components which can be used with Tooltip can accept onClick and handle focus(e.g. IconContainer)
